File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

void pyramid(int height);

int main(void)
{
    int height;

    do
    {
        height = get_int("Digite a altura: ");
    }
    while (height < 1 || height > 8);

    pyramid(height);
}

void pyramid(int height)
{
    for (int i = 0; i < height; i++)
    {
        for (int x = height - i; x > 1; x--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario/less/mario.py
from cs50 import get_int

while True:

    height = get_int('Digite a altura da pirâmide. O valor deve ser de 1 até 8: ')

    if height < 1 or height > 8:
        print('A altura da pirâmide deve ser um número de 1 até 8.')
        height = get_int('Digite a altura da pirâmide. O valor deve ser de 1 até 8: ')

    if height >= 1 or height <= 8:
        break

    else:
        print('A altura da pirâmide deve ser um número de 1 até 8.')
        height = get_int('Digite a altura da pirâmide. O valor deve ser de 1 até 8: ')

for i in range(height):
    print((height - 1 - i) * " ", end="")
    print((1 + i) * "#")
